NeSyPatterns/AS.der.hs

```diff
@@ -32,10 +32,10 @@ import Data.Data
 -- DrIFT command
 {-! global: GetRange !-}
 
--- | nodes are of form: ontology_term[id]
--- | both components are optional, but at least one must be present
+-- | nodes are of form: id : ontology_term
+-- | id is optional
 data Node = Node {
-    ontologyTerm :: (Maybe Id.Token),
+    ontologyTerm :: Id.Token,
```

Comment:
Why not IRI instead of Token?

Reply:
I remember @tillmo mentioning this should be done in a later task. Is this still true or can it done in this?

Reply:
I have a workaround in the colimit computation, so I guess it could stay. @tillmo what do you think?

Reply:
If you have a workaround for colimits, let us stay with Token here, and have a separate issue for changing it into IRI.